source/s3_meta_request.c

```diff
@@ -936,17 +941,6 @@ void aws_s3_meta_request_sign_request_default_impl(
         (void *)request,
         (void *)request->send_data.message);
 
-    if (request->send_data.signable == NULL) {
```

Comment:
nice job shuffling around these log statements and if-checks
makes more sense now
👍